capstone/src/main/java/com/google/univiz/SuggestionResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.google.univiz;
```

Comment:
According to the new packaging that @biancamacias is doing, this class should probably go into `api.representation` package. Or it could go into `univiz.scorecard`. Both seems fine to me.

Reply:
I think the representation package is probably better, just because this doesn't use nay scorecard functions/structures

Reply:
haha I will move it to scorecard to be in the same package as suggestiondata 👍